lib/shared/util.ts

```diff
@@ -11,4 +16,72 @@ export default class Util {
     static indexOf = indexOf;
     static sortBy = sortBy;
     static format = nodeUtil.format;
+    static request = request;
+    static path = path;
+    static fs = fs;
+    static childProcess = childProcess;
+
+    static getPackageJsonFromSource(url: string) {
+        let content: any;
+
+        return this.request({
+            method: "GET",
+            uri: url,
+            json: true,
+            resolveWithFullResponse: true,
```

Comment:
Why do we need the full response here if we'll only use the body part of it?

Reply:
resolveWithFullResponse was needed for the dynamic template discovery die to pagination, however, this is not the case anymore and we do not needed. As a mater of fact I will remove the unused method. The idea was to create a helper methods and to reuse them across the extension if we ever need to.